com.unity.cinemachine/Runtime/Debug/CinemachineDebug.cs

```diff
@@ -0,0 +1,149 @@
+using System;
+using UnityEngine;
+using System.Collections.Generic;
+using System.Text;
+#if UNITY_EDITOR && CINEMACHINE_UIELEMENTS
+using UnityEditor;
+using UnityEngine.UIElements;
+#endif
+
+namespace Unity.Cinemachine
+{
+    /// <summary>Manages onscreen positions for Cinemachine debugging output</summary>
+    static class CinemachineDebug
+    {
+        static List<StringBuilder> s_AvailableStringBuilders;
+
+#if CINEMACHINE_UNITY_IMGUI
+        /// <summary>Reserve an on-screen rectangle for debugging output.</summary>
+        /// <param name="client">The client caller.  This is used as a handle.</param>
+        /// <param name="text">Sample text, for determining rectangle size</param>
+        /// <param name="style">What style will be used to draw, used here for
+        /// determining rect size</param>
+        /// <returns>An area on the game screen large enough to print the text
+        /// in the style indicated</returns>
+        public static Rect GetScreenPos(Camera camera, string text, GUIStyle style)
```

Comment:
This is no longer used - remove the whole section and the CINEMACHINE_UNITY_IMGUI define from the asmdef